cli/rpm/header.go

```diff
@@ -93,28 +125,51 @@ func genRpmHeader(relPaths []string, cpioPath, compresedCpioPath string, ctx *co
 		{ID: tagPayloadDigestAlgo, Type: rpmTypeInt32, Value: []int32{int32(payloadDigestAlgo)}},
 	}...)
 
-	if !ctx.Tarantool.TarantoolIsEnterprise {
-		// add Tarantool dependency
-		tarantoolDepName := "tarantool"
-		flagGreaterOrEqual := int32(rpmSenseGreater | rpmSenseEqual)
+	var deps common.PackDependencies
 
-		minVersion := strings.SplitN(ctx.Tarantool.TarantoolVersion, "-", 2)[0]
-		maxVersion, err := common.GetNextMajorVersion(minVersion)
+	if !ctx.Tarantool.TarantoolIsEnterprise {
```

Comment:
This part is now common for both rpm and deb.
Simply realize `deps.Add()` function.